react/conf.py

```diff
@@ -0,0 +1,176 @@
+# -*- coding: utf-8 -*-
+#
+# Configuration file for the Sphinx documentation builder.
+#
+# This file does only contain a selection of the most common options. For a
+# full list see the documentation:
+# http://www.sphinx-doc.org/en/master/config
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = u'React'
+copyright = u'2018, Plone Community'
+author = u'Plone Community'
+
+# The short X.Y version
+version = u''
+# The full version, including alpha/beta/rc tags
+release = u'0.0.1-alpha'
+
+
+# -- General configuration ---------------------------------------------------
+
+# If your documentation needs a minimal Sphinx version, state it here.
+#
+# needs_sphinx = '1.0'
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = [
+]
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ['_templates']
+
+# The suffix(es) of source filenames.
+# You can specify multiple suffix as a list of string:
+#
+# source_suffix = ['.rst', '.md']
+source_suffix = '.rst'
+
+# The master toctree document.
+master_doc = 'index'
+
+# The language for content autogenerated by Sphinx. Refer to documentation
+# for a list of supported languages.
+#
+# This is also used if you do content translation via gettext catalogs.
+# Usually you set "language" from the command line for these cases.
+language = None
+
+# List of patterns, relative to source directory, that match files and
+# directories to ignore when looking for source files.
+# This pattern also affects html_static_path and html_extra_path .
+exclude_patterns = [u'_build', 'Thumbs.db', '.DS_Store']
+
+# The name of the Pygments (syntax highlighting) style to use.
+pygments_style = 'sphinx'
+
+
+# -- Options for HTML output -------------------------------------------------
+
+# The theme to use for HTML and HTML Help pages.  See the documentation for
+# a list of builtin themes.
+#
+html_theme = 'alabaster'
```

Comment:
IMO, alabaster is the worst choice for Sphinx themes due to poor navigability. Any other theme is better, including "classic" or "nature" (included) or https://github.com/rtfd/sphinx_rtd_theme

Reply:
Yes, "alabaster" is wrong :).
The docs are using the RTD theme with a custom CSS.

I guess the reason is the the old `config.py`

``
    html_theme = 'default'
``

And default is now alabaster for Sphinx.

Reply:
A lot of other trainings are also using 'alabaster', that's why I used it, switched to RTD now.

Reply:
Yeah, maybe some explanation, about this :)

Currently the training is build from within the /root of the repository. 
This means the local *conf.py* files are not used.

Every training contains a own `conf.py` because the idea is if you give training about one certain topic, you only want to show/give your students this part of the training docs.
It can be demotivating to show a student who is following a course about theming docs about 'hardcore' dev stuff.
The other reason that is done, is because we will/want to build and push the standalone to docs.training.org, so that we do not need to build all docs, if you update "your" training.

IMHO all of this should be added maybe to a "how to write" docs for training section ? 
I have the feeling that lots of this is not known or got forgotten ?

Reply:
AFAIK, it doesn't exist. There's no `contributing.*` in the root or a mention in the `README.rst`. You have to dig into this file: https://training.plone.org/5/mastering-plone/about_mastering.html#building-the-documentation-locally